### PR TITLE
krita: update to 4.4.0.

### PR DESCRIPTION
**Pull request by kartikynwa**
I have built both packages and can confirm they work perfectly at least for x86_64. The SeExpr module for Krita is working as well.

Should I test cross-building for `seexpr-krita`? I can't do it for `krita` cuz my computer is slow but can test the `seexpr-krita` if needed.

### CODE REVIEW

File: srcpkgs/seexpr-krita/template [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Template file for 'seexpr-krita'
+pkgname=seexpr-krita
+version=3.4.4.0
+revision=1
+wrksrc="seexpr-${version}"
+build_style=cmake
+configure_args="-DBUILD_DEMOS=OFF -DBUILD_TESTS=OFF \
```
Comment (Johnnynator) [inline srcpkgs/seexpr-krita/template:7]: No need to escape a newline in a quoted string. Also don't hard disable tests. add smth like
```
if [ "$XBPS_CHECK_PKGS" ]; then
        configure_args+=" -DBUILD_TESTS=ON"
else
        configure_args+=" -DBUILD_TESTS=OFF"
fi
```
after checksum and

```
do_check() {
        cd build
        # BlockTests requires llvm, we don't build it with llvm
        ctest -E 'BlockTests'
}

post_install() {
        [ -d "${DESTDIR}/usr/share/test/SeExpr2" ] && rm -r "${DESTDIR}/usr/share/test/SeExpr2" || true
}
```
 after pre_configure

File: srcpkgs/seexpr-krita/template [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	sed -i 's#share/SeExpr2/utils#lib/SeExpr2/utils#' src/utils/CMakeLists.txt
+}
+
+seexpr-krita-devel_package() {
+	short_desc+=" - development files"
+	depends="${sourcepkg}-${version}_${revision}"
+	pkg_install() {
+		vmove usr/include
+		vmove usr/share/cmake
+		vmove usr/share/pkgconfig
```
Comment (Johnnynator) [inline srcpkgs/seexpr-krita/template:28]: Add `vmove "usr/lib/*.so"`

File: srcpkgs/seexpr-krita/template [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Template file for 'seexpr-krita'
+pkgname=seexpr-krita
+version=3.4.4.0
+revision=1
+wrksrc="seexpr-${version}"
+build_style=cmake
+configure_args="-DBUILD_DEMOS=OFF -DBUILD_TESTS=OFF \
+ -DPYQT_SIP_DIR=/${py3_sitelib}/PyQt5/bindings"
+hostmakedepends="extra-cmake-modules"
+makedepends="qt5-tools-devel boost-devel python3-devel python3-PyQt5-devel python3-sip-devel"
```
Comment (Johnnynator) [inline srcpkgs/seexpr-krita/template:10]: Add `checkdepends="gtest-devel"`

**Review by Johnnynator — CHANGES_REQUESTED** [review]
seexpr doesn't cross compile ootb, either try to fix it or set it `nocross` and make the dep in krita conditional.

**Comment by kartikynwa** [comment]
@Johnnynator I tried cross-building it for `aarch64-musl`. Many of the errors were fixed by moving the `makedepends` to `hostmakedepends`. [This](http://ix.io/2BaS) is the template I have now. But I get [this](http://ix.io/2BaJ) error now that I don't have any idea about. Any pointers?

**Comment by kartikynwa** [comment]
@Johnnynator I put the `-DUSE_PYTHON=OFF` flag in `seexpr-krita`. It can now be cross-built (tested with armv6l-musl and armv7) and still has full functionality with `krita`.

**Pull request merged**